docs/tracing.rst

```diff
@@ -0,0 +1,144 @@
+.. _tracing:
+
+Tracing
+=======
+
+.. contents::
+    :depth: 3
+
+
+Overview
+--------
+
+It provides an instrument tracing wrapper function to use to instrument functions manually in pubtools-* projects.
+
+
+Usage
+.....................
+
+Set environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Following environment variables are used in the module:
+
+- ``OTEL_TRACING``: set ``true`` to enable instrument tracing, otherwise tracing is disabled.
+- ``OTEL_SERVICE_NAME``: required, set the value of the service.name resource attribute. It's
```

Comment:
At the moment I don't quite understand this, can you give me an example?  (Just telling me here and not putting into the docs.)  For example would we be setting this to "pub" ?

Reply:
"pub-hub" for Pub hub, and "pubd" (or pub-worker) for Pub worker. It can be as a filter field in field condition.